commit 9a87be24c4d520d465febd764b6c8190ecc1b1a2
Author: Trent Mick <trentm@gmail.com>
Date:   2019-07-19T10:01:46-07:00 (3 months ago)
    
    TRITON-1824 make mockcloud and sdc-system-tests builds upload to public builds area

diff --git a/tools/jr-manifest.json b/tools/jr-manifest.json
index e519695..1496b92 100644
--- a/tools/jr-manifest.json
+++ b/tools/jr-manifest.json
@@ -802,8 +802,7 @@
             "name": "sdc-system-tests",
             "labels": {
                 "release": true,
-                "mg": "sdc-system-tests",
-                "buildisprivate": true
+                "mg": "sdc-system-tests"
             }
         },
         {
@@ -949,7 +948,6 @@
                 "release": true,
                 "image": "mockcloud",
                 "mg": "mockcloud",
-                "buildisprivate": true,
                 "vm": true
             }
         },
